src/server/server.js

```diff
@@ -1,22 +1,34 @@
 import {SocketCluster} from 'socketcluster';
-import {auth0Secret} from './secrets';
 import path from 'path';
+import dotenv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
 // import os from 'os';
 
+// Import .env and expand variables:
+try {
+  const myEnv = dotenv.config({silent: true});
+  dotenvExpand(myEnv);
+} catch (e) {
+  console.warn(`Unable to load .env: ${e}`);
+}
+
 // const numCpus = os.cpus().length;
 export const options = {
-  authKey: new Buffer(auth0Secret, 'base64'),
```

Comment:
`const authKey = process.env.AUTH0_CLIENT_SECRET  && process.env.AUTH0_CLIENT_SECRET || 'BksPeQQrRkXhDrugzQDg5Nw-IInub9RkQ-pSWohUM9s6Oii4xoGVCrK2_OcUCfYZ'.`

Looks a little cleaner to define a variable above so we don't introduce much logic into the object creation. Plus it gets rid of the duplicate Buffer creation stuff